handlers/map-tactile-svg/map-svg.py

```diff
@@ -0,0 +1,292 @@
+        response = {
+            "request_uuid": contents["request_uuid"],
+            "timestamp": int(time.time()),
+            "renderings": []
+        }
+        try:
+            validator = jsonschema.Draft7Validator(
+                response_schema, resolver=resolver)
+            validator.validate(response)
+        except jsonschema.exceptions.ValidationError as error:
+            logging.error(error)
+            return jsonify("Invalid Preprocessor JSON format"), 500
+        logging.debug("Sending response")
+        return response
+
+    if "ca.mcgill.a11y.image.preprocessor.openstreetmap"\
+            not in preprocessor:
+        logging.debug("OSM Preprocessor data not present. Skipping ...")
+        response = {
+            "request_uuid": contents["request_uuid"],
+            "timestamp": int(time.time()),
+            "renderings": []
+        }
+        try:
+            validator = jsonschema.Draft7Validator(
+                response_schema, resolver=resolver)
+            validator.validate(response)
+        except jsonschema.exceptions.ValidationError as error:
+            logging.error(error)
+            return jsonify("Invalid Preprocessor JSON format"), 500
+        logging.debug("Sending response")
+        return response
+
+    dimensions = 700, 700
+    remove_streets = ["footway", "crossing", "steps"]
+    svg = draw.Drawing(dimensions[0], dimensions[1])
+
+    data = preprocessor["ca.mcgill.a11y.image.preprocessor.openstreetmap"]
+    if "streets" in data:
+        streets = data["streets"]
+        lat = data["bounds"]["latitude"]
+        lon = data["bounds"]["longitude"]
+        lon_min = lon["min"]
+        lat_min = lat["min"]
+        lon_max = lon["max"]
+        lat_max = lat["max"]
+        # Scale the lon/lat units to svg pixels equivalent.
+        scaled_longitude = dimensions[0] / (lon_max - lon_min)
+        scaled_latitude = dimensions[1] / (lat_max - lat_min)
+
+        colors = [
+                  "red",
+                  "blue",
+                  "springgreen",
+                  "deeppink",
+                  "orange",
+                  "purple",
+                  "cyan",
+                  "coral",
+                  "teal",
+                  "indigo",
+                  "lime",
+                  "chocolate",
+                  "magenta",
+                  "crimson",
+                  "deepskyblue",
+                  "greenyellow",
+                  "gold",
+                  "green",
+                  "aqua",
+                  "navy",
+                  "royalblue",
+                  "forestgreen",
+                  "dodgerblue"
+                  ]
+        # Draw the streets with svg.
+        checkPOIs = {}
+        g = draw.Group(data_image_layer="firstLayer", aria_label="Streets")
+        for i, street in enumerate(streets):
+            color = i
+            if i >= len(colors):
+                color = i % len(colors)
```

Comment:
Could probably just include this without check since if `i < len(colors)` then `i == i % len(colors)`.